src/platforms/apple/common/configuration/out-of-memory.mdx

```diff
@@ -12,7 +12,9 @@ The SDK might falsely report out-of-memory crashes when an app hangs, and the us
 
 This integration tracks out-of-memory (OOM) crashes based on heuristics. This feature is available for iOS, tvOS, and Mac Catalyst, works only if the application was in the foreground, and doesn't track OOMs for unit tests.
 
-When your application uses too much RAM, the operating system kills your app without a normal termination process. Your process doesn't receive a signal or any other type of information to ensure an OOM occurs. As a result, in the Apple SDK, we track out-of-memory crashes during the app start based on heuristics.
+When a typical unhandled error occurs, the Apple SDK writes a report to disk with the current state of your application, like the stack trace, tags, breadcrumbs, etc., before the app terminates. For OOMs, the operating system can kill your app without further notice, and the SDK can't write a report to disk.
+As a result, in the Apple SDK, we track out-of-memory crashes during the app start based on heuristics, and getting the state of the app when an OOM occurs is challenging. To provide the same state as for a typical crash, we would periodically need to store the app state to disk, causing a lot of IO, a tradeoff we are unwilling to accept. Therefore OOM events have less context than typical unhandled errors.
```

Comment:
I don't think we need to spell out out-of-memory crashes once we've defined it once on the page. 

```suggestion
As a result, in the Apple SDK, we track OOM crashes during the app start based on heuristics, but getting the state of the app when an OOM occurs is challenging. To provide the same state as we do for a typical crash, we would periodically need to store the app state to disk. This would cause a lot of I/O, which in turn could negatively affect the application's performance. To avoid this, OOM events have less context than typical unhandled errors.
```